clang/lib/Headers/__clang_cuda_math.h

```diff
@@ -345,4 +349,5 @@ __DEVICE__ float ynf(int __a, float __b) { return __nv_ynf(__a, __b); }
 #pragma pop_macro("__DEVICE_VOID__")
 #pragma pop_macro("__FAST_OR_SLOW")
 
+#endif // __CLANG_GPU_DISABLE_MATH_WRAPPERS
```

Comment:
some non-libm functions e.g. `__fadd_rd`, are included in the if/endif. Do you plan to add them to libm?

Reply:
Hm, good question. I added `powi` because that's a reasonable extension / renaming of `pown` that others seemed to have used. A double underscore doesn't quality for that I don't think. I don't think we could put that in the `libm` directly. What does that function do exactly?